webplugin/js/app/media/media-service.js

```diff
@@ -0,0 +1,46 @@
+Kommunicate.mediaService = {
+    capitalizeFirstCharacter: function (str) {
+        var firstCharRegex = /\S/;
+        return str.replace(firstCharRegex, function (m) {
+            return m.toUpperCase();
+        });
+    },
+    processMicClickedEvent: function () {
+        if (!('webkitSpeechRecognition' in window)) {
+            alert("browser do not support speech recogization");
+        } else {
+            var recognition = new webkitSpeechRecognition();
+            recognition.continuous = false; // The default value for continuous is false, meaning that when the user stops talking, speech recognition will end. 
+            recognition.interimResults = true; // The default value for interimResults is false, meaning that the only results returned by the recognizer are final and will not change. Set it to true so we get early, interim results that may change. 
+            finalTranscript = '';
+            recognition.lang = "en-us";
```

Comment:
@Suraj-Kumar @devashishmamgain Will this language option be in any way related to the localization setting that we have in our chat widget. Also, we will be creating an option to add a welcome message in the dashboard in some of the languages other than English. So, shouldn't this thing support at least those languages? Just a thought.

Reply:
got your point.  will check this.